baxter_sim_kinematics/src/arm_kinematics.cpp

```diff
@@ -245,8 +248,13 @@ bool Kinematics::loadModel(const std::string xml) {
  */
 bool Kinematics::readJoints(urdf::Model &robot_model) {
   num_joints = 0;
+  #if ROS_VERSION_MINIMUN(1, 14, 0)  // Melodic
```

Comment:
I'm so sorry, but my original comment was typo.
`ROS_VERSION_MINIMUN` -> `ROS_VERSION_MINIMUM`
can you correct this?